src/scripting/lua_widget_attributes.cpp

```diff
@@ -47,10 +52,17 @@
 #include <utility>
 #include <vector>
 
-
 static lg::log_domain log_scripting_lua("scripting/lua");
 #define ERR_LUA LOG_STREAM(err, log_scripting_lua)
 
+#define INVALIDATE_LAYOUT \
```

Comment:
Make this `INVALIDATE_LAYOUT()`. Since it's a macro that does stuff, like calling a function, I prefer it to _look_ like a function.